src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -284,6 +284,14 @@ def create_live_stream(key):
     }
 
 
+def wait_medialive_channel_is_created(channel_id):
+    """Wait until medialive channel is created."""
```

Comment:
```suggestion
    """Wait until medialive channel is created.
    
    Poll for 30 attempts with 1 second delay.
    """
```

Maybe I missed it, but shouldn't we add a test with a raised exception ?

Reply:
Here I don't really see the interest. In the `start-live` endpoint api why not.

Reply:
Yes, that's what I mean :)

Reply:
if AWS raises an exception then we return a 500 and the frontend has to deal with it? so nothing to test on the back I would say?

Reply:
Anyway we are looking to refactor this long polling so let's merge the refactoring like this to unlock @carofun's PR and work on the asynchronous refactoring in a next PR.